pgcontents/tests/test_pgmanager.py

```diff
@@ -84,6 +84,79 @@ def test_modified_date(self):
         renamed = cm.get(new_path)
         self.assertGreater(renamed['last_modified'], saved['last_modified'])
 
+    def test_rename_directory(self):
+        cm = self.contents_manager
```

Comment:
This can be made a fair amount cleaner using methods inherited from the parent test class I think.  I'd also like to break this up into smaller chunks if we can.  Let's talk when you get in today.